pymc3/tests/test_distributions_random.py

```diff
@@ -174,12 +174,7 @@ def get_random_variable(self, shape, with_vector_params=False, name=None):
                         # in the test case parametrization "None" means "no specified (default)"
                         return self.distribution(name, transform=None, **params)
                     else:
-                        ndim_supp = self.distribution.rv_op.ndim_supp
-                        if ndim_supp == 0:
-                            size = shape
-                        else:
-                            size = shape[:-ndim_supp]
-                        return self.distribution(name, size=size, transform=None, **params)
+                        return self.distribution(name, shape=shape, transform=None, **params)
```

Comment:
Remember, we are **not** using `shape` internally.

Reply:
Yes. I reverted these to `shape` because I was lazy. They should be refactored to use `shape`, but for many test that means we have to `pytest.mark.parametrize` with both `shape` and `size`, because we often use the `shape` for asserts.